Generals/Code/GameEngine/Source/Common/INI/INI.cpp

```diff
@@ -1917,8 +1917,8 @@ Bool INI::isDeclarationOfType( AsciiString blockType, AsciiString blockName, cha
 		retVal = false;
 	}
 
-	while (strlen(tempBuff)) {
-		retVal = retVal && isspace(tempBuff[0]);
+	while (*tempBuff) {
+		retVal = retVal && isspace(*tempBuff);
```

Comment:
If I am not mistaken this can be further optimized as

```cpp
while (*tempBuff && retVal) {
	retVal = isspace(*tempBuff);
	++tempBuff;
}
```

Basically just checks if the remaining chars are just whitespace and nothing else

Reply:
Good catch!